pkg/entrypoint/entrypointer.go

```diff
@@ -77,7 +75,7 @@ type Entrypointer struct {
 	// OnError defines exiting behavior of the entrypoint
 	// set it to "stopAndFail" to indicate the entrypoint to exit the taskRun if the container exits with non zero exit code
 	// set it to "continue" to indicate the entrypoint to continue executing the rest of the steps irrespective of the container exit code
-	OnError string
+	OnError v1beta1.OnErrorType
```

Comment:
I'd keep the entrypoint types decoupled from the API types as much as possible - the entrypoint needs to support a superset of states across all versions. e.g. if there's a new OnError state in a v1alpha2 API, the entrypoint will need to add it (and it'll be easier to support if we don't have to add it to stable APIs and deal with potential deprecation policies).

Reply:
Yep, this makes sense! I have converted converted the entrypoint types back. Please take a look!